Source/Classes/PINRemoteImageManager.m

```diff
@@ -1556,6 +1557,32 @@ - (NSString *)cacheKeyForURL:(NSURL *)url processorKey:(NSString *)processorKey
     return [self cacheKeyForURL:url processorKey:processorKey resume:NO];
 }
 
+-(NSString *) hashCacheKey_removeMeIOS13:(NSString *) string {
```

Comment:
nit: to match code style elsewhere, add a space after the - and remove the ones after the )

Reply:
done!